src/orders/model/OrderPatchRequest.ts

```diff
@@ -0,0 +1,6 @@
+import Order from './Order';
+import { RecursivePartial } from '../../model/RecursivePartial';
+
+export default interface OrderPatchRequest extends RecursivePartial<Order> {
+  actionId?: string;
```

Comment:
This is my fault for not bringing this up in code review but we need one more optional boolean field here titled `channelSync`. That is the field we will use to sync changes back to Shopify when true on the REST API.

Can you also add this in OrderUpdateRequest.ts as well?